src/main/java/webserver/RequestHandler.java

```diff
@@ -26,21 +26,33 @@ public void run() {
             // TODO 사용자 요청에 대한 처리는 이 곳에 구현하면 된다.
             BufferedReader br = new BufferedReader(new InputStreamReader(in, "UTF-8"));
 
-            HttpRequestHeader httpRequestHeader = new HttpRequestHeader(br);
-            HttpResponseBuilder httpResponseBuilder = new HttpResponseBuilder();
+            HttpRequest httpRequest = new HttpRequest(br);
 
-            String returnUrl = httpRequestHeader.getValueByName("returnUrl");
+
+            String returnUrl = httpRequest.getValueByName("returnUrl");
 
             DataOutputStream dos = new DataOutputStream(out);
 
             File f = new File(templatePath + returnUrl);
             if (!f.exists()) {
                 f = new File(staticPath + returnUrl);
             }
+            if (!f.exists()) {
+                returnUrl = "/util/error.html";
```

Comment:
```suggestion
                returnUrl = "/error.html";
```

URL은 일종의 리소스라고 볼 수 있는데 util 같은 경로는 이상합니다. 없는 편이 좋을 것 같아요.